### ISSUE TITLE
I cannot seem to get these to show

### ISSUE THREAD

**Issue by duro** [andys8/vscode-jest-snippets#3]
My settings file looks like this:

```
{
  "workbench.colorTheme": "Charcoal Oceanic Next",
  "workbench.colorCustomizations": {
    "statusBar.background": "#303030",
    "statusBar.noFolderBackground": "#222225",
    "statusBar.debuggingBackground": "#511f1f",
    "editor.lineHighlightBackground": "#2E3337",
    "editorBracketMatch.border": "#575B5F"
  },
  "window.zoomLevel": 1,
  "editor.fontSize": 16,
  "editor.fontFamily": "Menlo, Consolas, 'DejaVu Sans Mono', monospace",
  "editor.tabSize": 2,
  "prettier.eslintIntegration": true,
  "prettier.singleQuote": true,
  "prettier.semi": false,
  "editor.formatOnSave": true,
  "colorize.hide_current_line_decorations": false,
  "colorize.languages": [
    "javascript",
    "css",
    "sass",
    "scss",
    "less",
    "pcss",
    "sss",
    "stylus",
    "xml",
    "svg"
  ],
  "workbench.startupEditor": "newUntitledFile",
  "autoimport.filesToScan": "**/*.{ts,tsx,js}",
  "files.associations": {
    "*.js": "javascriptreact"
  },
  "autofilename.extensions.trim": ["js"],
  "editor.snippetSuggestions": "top",
  "editor.tabCompletion": true
}
```

But whenever I try and enter in one of the snippet commands, all I get back are irrelevant Intellisense suggestion.

Any ideas?

**Comment by andys8** [comment]
Hi, just checked it to be working with the current version of vscode:
![image](https://user-images.githubusercontent.com/13085980/29842196-686de7aa-8d08-11e7-87e3-6fc0416d98f9.png)

Trying to help you, but to be honest, I'm guessing:
* Are you working on a javascript file (defined [here](https://github.com/andys8/vscode-jest-snippets/blob/master/package.json#L26))?
* Do you have trouble with other snippet extensions or did you try custom user snippets instead?
* Could other extensions be interfering?

## Update

Okay, looking at your settings, I think redefining the file associations is the cause. If this settings it won't show up anymore because "javascriptreact" is something different. 
```json
"files.associations": {
    "*.js": "javascriptreact"
  }
```
![image](https://user-images.githubusercontent.com/13085980/29842599-a0814942-8d09-11e7-8b8f-90d68b4bb508.png)

What's the cause for this setting and is it popular? Could make sense to adapt the extension.

**Comment by andys8** [comment]
@duro Updated the definition and released `1.0.3`. Tested and it should work now. Tell me if it's solved for you ;)
